src/tls_wrap.cc

```diff
@@ -881,6 +889,90 @@ int TLSWrap::SelectSNIContextCallback(SSL* s, int* ad, void* arg) {
 #endif  // SSL_CTRL_SET_TLSEXT_SERVERNAME_CB
 
 
+#ifdef OPENSSL_PSK_SUPPORT
+
+unsigned int TLSWrap::PskServerCallback(SSL *ssl,
```

Comment:
@stephank depends, if you will implement it in `SecureContext` - callbacks will have to be called on JS `SecureContext` instance too. This seems to be rather unusual in terms of what APIs we currently have.